src/gui/MainWindow.cpp

```diff
@@ -304,7 +304,7 @@ void MainWindow::finalize()
 	project_menu->addSeparator();
 
 	project_menu->addAction(embed::getIconPixmap("project_import"), tr("Import..."),
-		this, &MainWindow::onImportProject);
+		keySequence(Qt::CTRL, Qt::Key_I), this, &MainWindow::onImportProject);
```

Comment:
Unfortunately, this only works with Qt 6.3 or higher. For Qt 5.x or 6.0 to 6.2, it should be the last argument.

Reply:
Try using the `addAction` lambda I added in the Qt6 PR:
```cpp
addAction(project_menu, embed::getIconPixmap("project_import"), tr("Import..."),
	keySequence(Qt::CTRL, Qt::Key_I), &MainWindow::onImportProject);
```

Several other lines in this method are already using it.